components/brave_vpn/brave_vpn_service.cc

```diff
@@ -688,6 +748,30 @@ void BraveVpnService::OnPrepareCredentialsPresentation(
 #endif
 }
 
+void BraveVpnService::ScheduleSubscriberCredentialRefresh() {
+  if (subs_cred_refresh_timer_.IsRunning())
+    subs_cred_refresh_timer_.Stop();
+
+  const auto expiration_time = GetExpirationTime(local_prefs_);
+  if (!expiration_time)
+    return;
+
```

Comment:
Maybe silly - but we could avoid `*expiration_time - base::Time::Now()` logic happening twice if we stored in a variable
ex:
```
auto expiration_time_delta = *expiration_time - base::Time::Now();
VLOG(2) << "Schedule subscriber credential fetching after " << expiration_time_delta;
subs_cred_refresh_timer_.Start(
      FROM_HERE, expiration_time_delta,
```

Reply:
Updated :)